apis/cache/v1beta1/redis_types_test.go

```diff
@@ -0,0 +1,45 @@
+/*
+Copyright 07 November 2019 Muvaffak Onus.
```

Comment:
Are you particularly proud of this specific test file? ;)

(This needs to be changed to "The Crossplane Authors" similar to the other license headers.)

Reply:
No, not particularly :) I guess Goland defaulted to that header automatically when I created the file. Thanks for catching that!